image-search/ViewControllers/ViewController.swift

```diff
@@ -40,14 +46,24 @@ class ViewController: UIViewController {
     
         @IBAction func alterText(_ sender: Any) {
             
+            activityIndicatorIndicator.center = self.view.center
+            activityIndicatorIndicator.style = UIActivityIndicatorView.Style.white
+            view.addSubview(activityIndicatorIndicator)
+            
+           
             let album = getNewAlbum()
             labelNameMusic.text = album.recordLabel
             labelNameArtist.text = album.artistName
             label.text = album.albumName
             
             
+            activityIndicatorIndicator.startAnimating()
```

Comment:
You can start this indicator on the beginning of function, before start setting anything of albums.

Reply:
Changed